src/index.html

```diff
@@ -1,13 +1,50 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body class="page__product-cards">
+    <form class="card" data-qa="card">
+      <img
+        class="card__photo"
+        src="images/imac.jpeg"
+        alt="Imac"
+      >
+      <p class="card__name">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</p>
+
+      <p class="card__num">Код товара: 195434</p>
+      <div class="card__stars-line">
+        <div class="stars-group">
+          <div class="stars-group__star"></div>
+
+          <div class="stars-group__star"></div>
+
+          <div class="stars-group__star"></div>
+
+          <div class="stars-group__star"></div>
+
+          <div class="stars-group__star"></div>
+        </div>
+        <span class="card__otz">Отзывов: 5</span>
+      </div>
+      <div class="card__price">
+        <span class="name">Цена:</span>
+
+        <span class="cost">69 999 грн</span>
```

Comment:
these should not be BEM-blocks, make them elements of your `card` block

Reply:
done!